### PR TITLE
Release v.1.25.5

### PR DESCRIPTION
**Pull request by keriat**
Generally I've merged initial repo changes.

But alongside list of minor things was done too:

- peer-dependencies inside ```package.json``` were configured right way
- yarn bumped to v.2.* (berry)
- 70% of dependencies had minor updates, so they were upgraded
- local imports I've masked with the ts-path
- tsconfig.json was updated as well
- couple of styles inside Onboard.svelte were dropped since the build process was breaking each time
- provider for the Ledger wallet was totally outdated (discontinued), so I've switched dependency to the tenant package and fixed the connection

Now @popzxc, I'd appreciate small quick review to proceed with publishing and testing it on zkWallet

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -16,4 +16,4 @@ module.exports = {
     '@typescript-eslint/explicit-function-return-type': 'off',
     '@typescript-eslint/no-empty-function': 'off'
   }
-}
+}
```
Comment (popzxc) [inline .eslintrc.js:29]: Nit: Missing newline.

File: package.json [hunk 2]
```diff
@@ -26,57 +26,73 @@
...
     "fortmatic": "^2.2.1",
     "generator-svelte": "2.2.9",
     "hdkey": "^2.0.1",
     "node-fetch": "^2.6.1",
-    "regenerator-runtime": "^0.13.7",
-    "squarelink": "^1.1.4",
-    "trezor-connect": "^8.1.9",
-    "walletlink": "^2.0.2",
-    "web3-provider-engine": "^15.0.4"
+    "prettier-plugin-svelte": "^2.2.0",
```
Comment (popzxc) [inline package.json:83]: Shouldn't this be in `devDependencies`?

File: rollup.config.js [hunk 3]
```diff
@@ -81,6 +83,7 @@ export default {
     'web3-provider-engine/subproviders/hooked-wallet',
     'web3-provider-engine/subproviders/rpc',
     'web3-provider-engine/subproviders/subscriptions',
-    'web3-provider-engine/subproviders/filters'
+    'web3-provider-engine/subproviders/filters',
+    'eth-provider'
   ]
-}
+}
```
Comment (popzxc) [inline rollup.config.js:91]: Nit: Missing newline.

File: src/components/Modal.svelte [hunk 4]
```diff
@@ -140,4 +145,4 @@
       </div>
     {/if}
   </section>
-</aside>
+</aside>
```
Comment (popzxc) [inline src/components/Modal.svelte]: Nit: Missing newline (please check everywhere and maybe add it to the lint checks).

File: src/elements/Button.svelte [hunk 5]
```diff
@@ -14,9 +15,10 @@
     padding: 0.55em 1.4em;
     cursor: pointer;
     color: #4a90e2;
-    font-family: inherit;
-    transition: background 150ms ease-in-out;
+      transition: background 150ms ease-in-out;
```
Comment (popzxc) [inline src/elements/Button.svelte]: Nit: Inconsistent alignment.

**Review by popzxc — DISMISSED** [review]

File: LICENSE [hunk 6]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 Blocknative Corp
+Copyright (c) 2020 Matter Labs Corp
```
Comment (popzxc) [inline LICENSE]: I don't think we can easily get rid of the copyright. The right statement will be `Copyright (c) 2020 Matter Labs, Blocknative Corp`. Also, Matter Labs is not a corp anyways.

File: LICENSE [hunk 7]
```diff
@@ -14,8 +14,8 @@ copies or substantial portions of the Software.
 
 THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFINGEMENT. IN NO EVENT SHALL THE
```
Comment (popzxc) [inline LICENSE]: The original word was OK, why would you change it?

File: package.json [hunk 8]
```diff
@@ -90,6 +88,7 @@
   "peerDependencies": {
     "@babel/core": "*",
     "@babel/runtime": "*",
+    "@ledgerhq/hw-transport-webusb": "^5.51.0",
```
Comment (popzxc) [inline package.json]: Why did you make it a peer dependency? It's kinda weird that `hw-app-eth` is a common dependency, and this one is a peer one.

**Review by popzxc — CHANGES_REQUESTED** [review]

**Review by popzxc — DISMISSED** [review]

**Comment by keriat** [comment]
Abandoned in favor of the source-library.

**Pull request closed without merging**